### PR TITLE
CB-31071: prep for 3.7.0 release

### PR DESCRIPTION
**Pull request by dseidel-b9**

### CODE REVIEW

**Comment by dseidel-b9** [comment]
Holding this until the AMSI changes are merged to `release-3.7.0`, then we can review it again in toto,

**Comment by dseidel-b9** [comment]
AMSI changes are merged, and we're ready to review this for merging to master.

File: README.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+We support pre-built RPM via our [User eXchange (Jive)](https://community.carbonblack.com/community/developer-relations) 
 and via email to dev-support@carbonblack.com. 
 
 ## Raw Sensor Events 
 
-We have seen a performance impact when exporting all raw sensor events onto the enterprise bus. We do not recommend
-exporting all the events. The performance impacts are seen when the events are broadcast on the bus, by enabling the
-"DatastoreBroadcastEventTypes". We recommend that at most, only process and netconn events be broadcast on the event
+We have seen a performance impact when exporting all raw sensor events onto the enterprise bus by setting
+"DatastoreBroadcastEventTypes=True" in the EDR configuration (more onm this below). We do not recommend exporting all
```
Comment (sjones-github) [inline README.md:25]: ...more onm this below... appears to be a typo.

Comment (dseidel-b9) [reply README.md:25]: Ah yes indeed.

**Comment by dseidel-b9** [comment]
@sjones-github typo was fixed before I merged

**Pull request merged**